ca/certificate-authority.go

```diff
@@ -602,9 +601,7 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(ctx context.Context, csr x5
 
 	// Submit the certificate to any configured CT logs
 	go func() {
-		pubCtx, cancel := context.WithTimeout(context.Background(), ca.GRPCTimeout)
-		defer cancel()
-		_ = ca.Publisher.SubmitToCT(pubCtx, certDER)
+		_ = ca.Publisher.SubmitToCT(ctx, certDER)
```

Comment:
you still need a new context.Background() here, because the IssueCertificate deadline is shorter than the submit deadline